### DISCUSSION TITLE
Can you run Gnu Guix on Crouton at ChromeOS version 105.0.5195.134 ?

### DISCUSSION THREAD

**Discussion by IbrutoI** [dnschneid/crouton#5080] [General]
```
Please paste the output of the following command here: sudo edit-chroot -all
```
sudo edit-chroot -all
#### Please describe your issue:
Hi, was just wondering if it is possible to run Gnu Guix on crouton that's all

**Comment by IbrutoI** [comment]
Btw if it's important, i downgraded to chromeos version 105.0.5195.134

**Comment by CroutonIsFun** [comment]
This tab is for filing actual issues with crouton, not general discussions. Questions like this belong in the discussion section.

Anyhow, I don't think the ChromeOS version would have any effect on downloading and installing Guix within a crouton chroot. All you can do is try.

**Comment by IbrutoI** [comment]
oh alright, thanks
